packages/plugins/src/components/plugin-context/index.tsx

```diff
@@ -39,13 +48,13 @@ export const withPluginContext = (
 ) =>
 	createHigherOrderComponent( ( OriginalComponent ) => {
```

Comment:
Should we deprecate this?

Reply:
That's the plan, but first, I need to remove all usages from the core. It would be odd for core to log deprecated messages :)